porch/apiserver/Dockerfile

```diff
@@ -21,30 +21,18 @@ WORKDIR /go/src/github.com/GoogleContainerTools/kpt
 # and so that source changes don't invalidate our downloaded layer
 
 COPY go.mod go.sum ./
+COPY porch/go.mod porch/go.sum porch/
 COPY porch/api/go.mod porch/api/go.sum porch/api/
-COPY porch/apiserver/go.mod porch/apiserver/go.sum porch/apiserver/
-COPY porch/controllers/go.mod porch/controllers/go.sum porch/controllers/
-COPY porch/engine/go.mod porch/engine/go.mod porch/engine/
-COPY porch/repository/go.mod porch/repository/go.sum porch/repository/
-COPY porch/func/go.mod porch/func/go.sum porch/func/
 
 RUN echo "Downloading root modules ..." \
  && go mod download
+RUN echo "Downloading porch modules ..." \
```

Comment:
I'm surprised this one happens before the API module ...

Reply:
I don't think the order matters when we're doing go mod download.  Any preference?  The current order is alphabetical (sort of)